external-crates/move/tools/move-cli/src/sandbox/cli.rs

```diff
@@ -43,9 +43,8 @@ pub enum SandboxCommand {
         /// Manually specify the publishing order of modules.
         #[clap(
             long = "override-ordering",
-            takes_value(true),
-            multiple_values(true),
-            multiple_occurrences(true)
+            num_args(1..),
```

Comment:
Maybe it's late and I am missing something, but how does `num_args(1..)` work with `Option`

Reply:
the num_args just indicates the number or arguments it accepts when the flag is present, if its never present than Option will still result in None